yarn.lock

```diff
@@ -767,7 +767,7 @@
   dependencies:
     "@phosphor/algorithm" "^1.2.0"
 
-"@phosphor/widgets@^1.5.0":
+"@phosphor/widgets@^1.9.3":
```

Comment:
@marcdumais-work I've tried to run the complete check and seems to look fine, but it would be good if you double check after me 🙏

Reply:
Ok, will try to do so later today. 

BTW, Wayne sent an email taking about the new IP process for 3rd party dependencies - it looks based on the experimental process we've been using and all projects will be using it soon. They are working on automating the checks as well, which will be very nice :)

Reply:
I just did the complete check and I confirm that everything looks good.

Reply:
thank you Marc!